### PR TITLE
Fix site_level dosctring

### LINKED ISSUE #781: Docstring example missing ending backticks
Resulting in the docs [incorrectly rendering](https://open-aims.github.io/ADRIA.jl/dev/API/#ADRIA.metrics.loc_trajectory-Union{Tuple{A},%20Tuple{N},%20Tuple{T},%20Tuple{D},%20Tuple{Any,%20YAXArrays.Cubes.YAXArray{D,%20T,%20N,%20A}}}%20where%20{D,%20T,%20N,%20A}) the example

https://github.com/open-AIMS/ADRIA.jl/blob/f236fb66145e2002b048a76454cffeb8aa39d8de/src/metrics/site_level.jl#L26-L58

### PR DESCRIPTION
**Pull request by Zapiano**
Close #781

### CODE REVIEW

File: src/metrics/site_level.jl [hunk 1]
```diff
@@ -34,6 +34,7 @@ using Statistics
 
 rs = ADRIA.load_results("some results")
 tac = ADRIA.metrics.total_absolute_cover(rs)
+```
```
Comment (ConnectedSystems) [inline src/metrics/site_level.jl:37]: Ending backticks are in the wrong place.

Next few lines are also example code.

**Comment by ConnectedSystems** [comment]
Thanks for doing this!

File: src/metrics/site_level.jl (hunk 1 above)
Comment (Zapiano) [reply src/metrics/site_level.jl]: Oh, my bad. Ready for another review now.

**Review by ConnectedSystems — APPROVED** [review]
Thanks!

**Pull request merged**